hw/ip/keymgr/data/keymgr_testplan.hjson

```diff
@@ -6,13 +6,13 @@
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
+                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson"]
   entries: [
     {
       name: smoke
       desc: '''
             Basic smoke test accessing a major datapath within the keymgr.
```

Comment:
Also, this line was just supposed to be just a generic description. I was hoping for people to update it based on what the test will do for that DUT...

Reply:
Added brief description of the smoke test. Thanks